server/manager/src/main/java/org/apache/accumulo/manager/upgrade/Upgrader12to13.java

```diff
@@ -363,8 +364,18 @@ private void validateEmptyZKWorkerServerPaths(ServerContext context) {
         List<String> children = zr.getChildren(zkRoot + serverPath);
         for (String child : children) {
           if (child.contains(":")) {
-            throw new IllegalStateException("Found server address at " + serverPath + "/" + child
-                + ". Was expecting either a resource group name or nothing. Stop any referenced servers.");
+            String childPath = zkRoot + serverPath + "/" + child;
+            if (zr.getChildren(childPath).isEmpty()) {
+              // child is likely host:port and is an empty directory. Since there
+              // is no lock here, then the server is likely down (or should be).
+              // Remove the entry and move on.
+              context.getZooSession().asReaderWriter().recursiveDelete(childPath,
+                  NodeMissingPolicy.FAIL);
```

Comment:
It might be okay to use NodeMissingPolicy.SKIP instead... if something else deleted the lock node in the interim, that might be okay?

Reply:
Changed in 9f70857